src/model/resources/fusion/fusion_maintenance.jl

```diff
@@ -0,0 +1,33 @@
+# higher level function
+function fusion_maintenance_adjust_parasitic_power!(EP, rs::Vector{<:AbstractResource})
+    @debug "Maintenance modifies fusion parasitic power"
+
+    FUSION = ids_with(rs, fusion)
+    MAINTENANCE = ids_with_maintenance(rs)
+
+    for y in intersect(FUSION, MAINTENANCE)
+        resource_component = resource_name(rs[y])
+        reactor = FusionReactorData(rs, y)
+        _fusion_maintenance_parasitic_power_adjustment!(EP, resource_component, reactor)
+    end
+end
+
+# lower level function
+function _fusion_maintenance_parasitic_power_adjustment!(
+        EP::Model, resource_component, reactor::FusionReactorData)
+    passive = reactor.parasitic_passive_fraction
+    reduction_factor = reactor.maintenance_remaining_parasitic_power_fraction
+    η = reactor.eff_down
+    cap_size = reactor.component_size
```

Comment:
```suggestion
    component_size = reactor.component_size
```
to follow same convention as in `fusion.jl`

And the same for `get_from_model` -> `from_model`